src/test/java/com/rakovets/course/java/core/practice/string/StringUtilTest.java

```diff
@@ -0,0 +1,301 @@
+                    Arguments.of("alekseev aleksei", "AA"),
+                    Arguments.of("alekseev", "A")
+            );
+        }
+
+        @ParameterizedTest(name = "(initials) Text: {0}")
+        @MethodSource("provideArgumentsForInitials")
+        void initialsTest(String text, String expected) {
+            String actual = StringUtil.initials(text);
+
+            assertEquals(expected, actual);
+        }
+
+        static Stream<Arguments> provideArgumentsForEqualArrays() {
+            return Stream.of(
+                    Arguments.of(new String[]{"1", "2", "3", "4", "5"},
+                                 new String[]{"5", "4", "3", "2", "1"}, true),
+                    Arguments.of(new String[]{"Hello", "2", "3", "2", "5"},
+                                 new String[]{"5", "2", "3", "2", "Hello"}, true),
+                    Arguments.of(new String[]{"Hello", "2", "3", "2", "5"},
+                                 new String[]{"5", "2", "0", "2", "Hello"}, false),
+                    Arguments.of(new String[]{"1", "2", "3", "4", "5", "6"},
+                                 new String[]{"5", "4", "3", "2", "1"}, false),
+                    Arguments.of(new String[]{""},
+                                 new String[]{""}, true)
+            );
+        }
+
+        @ParameterizedTest(name = "(EqualArrays) Array1: {0}. Array2: {1}")
+        @MethodSource("provideArgumentsForEqualArrays")
+        void equalArraysTest(String[] array1, String[] array2, boolean expected) {
+            boolean actual = StringUtil.equalArrays(array1, array2);
+
+            assertEquals(expected, actual);
+        }
+
+        static Stream<Arguments> provideArgumentsForGetAllDigits() {
+            return Stream.of(
+                    Arguments.of("2b0c2e1", "2021"),
+                    Arguments.of("1_000_000", "1000000"),
+                    Arguments.of("", ""),
+                    Arguments.of(" ", "")
+            );
+        }
+
+        @ParameterizedTest(name = "(AmountDigits) Text: {0}")
+        @MethodSource("provideArgumentsForGetAllDigits")
+        void amountDigitsTest(String text, String expected) {
+            String actual = StringUtil.getAllDigits(text);
+
+            assertEquals(expected, actual);
+        }
+
+        static Stream<Arguments> provideArgumentsForDeleteExcessSymbols() {
+            return Stream.of(
+                    Arguments.of("AAAAbbbbbCCCCCddddDd", "AbCdD"),
+                    Arguments.of("abcdeeef", "abcdef"),
+                    Arguments.of("", ""),
+                    Arguments.of("   ", " ")
+            );
+        }
+
+        @ParameterizedTest(name = "(DeleteExcessSymbols) Text: {0}")
+        @MethodSource("provideArgumentsForDeleteExcessSymbols")
+        void DeleteExcessSymbolsTest(String text, String expected) {
+            String actual = StringUtil.deleteExcessSymbols(text);
+
+            assertEquals(expected, actual);
+        }
```

Comment:
"Violation: There aren't correct indentations for inner code
Solution: Add correct indentations for inner code"